src/main/resources/plugin.yml

```diff
@@ -58,3 +61,8 @@ permissions:
     children:
       justchat.click: true
       justchat.hover: true
+  justchat.command:
+    description: "Allows for usage of justchat commands"
+    default: op
+    children:
+      justchat.reload: true
```

Comment:
so my understanding is, if there's another subcommand added and I grant justchat.command permission, all players will be able to reload the plugin? This would make more sense if justchat.reload had child permission justchat.command, since you need permission for the command to use the subcommand at all